values.yaml

```diff
@@ -685,5 +685,10 @@ csi:
     # annotations to apply to the serviceAccount.
     annotations: {}
 
+  # Used to define custom readinessProbe settings
+  readinessProbe: {}
+  # Used to enable a livenessProbe for the pods.
+  livenessProbe: {}
```

Comment:
The defaults you defined up front should be in the `values.yaml` file. This makes them easier to discover by users instead of going to the website or looking at the source code.